server/package.json

```diff
@@ -34,6 +34,7 @@
     "keycloak-connect": "11.0.2",
     "lodash": "4.17.21",
     "massive": "6.8.0",
+    "morgan": "^1.10.0",
```

Comment:
we should lock this version before merge (or in a subsequent PR)

Reply:
👍